Unity-Technologies/Graphics
# Updates for Linux Vulkan tests for Universal and HDRP

# Purpose of this PR

> Gets Universal and HDRP test pipelines for Linux Vulkan into a better state.  There are still legitimate failures for both pipelines and for some reason the HDRP Standalone tests are capturing black screenshots for Linux Vulkan.  I also enabled split build and run for these pipelines.

> There is also the start of enabling UWP test for Universal as well.  This still needs work.

# Testing status
## Manual Tests
> I have run the tests multiple times through Yamato as well as existing pipelines to ensure they are unaffected.

## Automated Tests
> I didn't add any new tests.

## Links
**Yamato**: https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/tests%252Fupdate-refimages

# Comments to reviewers
> This is targeted towards trunk with the latest Graphics repo.  I started verifying the tests against 2020.1 but there are some blocking issues there that need to get investigated by the Desktop Graphics team.  The pipelines themselves should be fine without further changes once those issues are resolved.
